modules/calibration/data/dev_kit_advanced_3lidar/velodyne_dag/velodyne16.dag

```diff
@@ -0,0 +1,57 @@
+module_config {
+    module_library : "/apollo/bazel-bin/modules/drivers/robosense/librobosense_driver_component.so"
+    components {
+        class_name : "RobosenseComponent"
+        config {
+          name : "RS16_left_Driver"
+          config_file_path : "/apollo/modules/drivers/velodyne/conf/rs16_left.pb.txt"
+        }
+    }
+
+    components {
+        class_name : "RobosenseComponent"
```

Comment:
Also why it is Velodyne16 DAG but loading RobosenseComponent? Try to make the file name and actual content match.

Reply:
same for other files.

Reply:
This dag name is linked to HMI MODE of dev kit close loop, which I would not change. To make sure that tab on dreamview works, I have to use this one for robosense lidar

Reply:
> This dag name is linked to HMI MODE of dev kit close loop, which I would not change. To make sure that tab on dreamview works, I have to use this one for robosense lidar
better rename folder_name from velodyne_dag/ to lidar_dag/   rename file_name frome velodyne16.dag to lidar.dag